src/components/common/Members/filterMembers.ts

```diff
@@ -1,55 +1,63 @@
 import { FormValues } from '~common/ColonyMembers/MembersFilter';
 import { VerificationType } from '~common/ColonyMembers/MembersFilter/filtersConfig';
 
-import { Contributor, Watcher } from '~types';
+import { Address, Contributor, Watcher } from '~types';
+import { isUserVerified } from '~utils/verifiedUsers';
 
 const filterMemberBySearchTerm = (
   member: Contributor | Watcher,
-  searchTerm?: string,
+  searchTerm: string,
 ) => {
   return (
-    searchTerm &&
-    (member?.user?.profile?.displayName
+    member?.user?.profile?.displayName
       ?.toLowerCase()
       .includes(searchTerm.toLowerCase()) ||
-      member?.user?.walletAddress
-        ?.toLowerCase()
-        .includes(searchTerm.toLowerCase()) ||
-      member?.user?.name.toLowerCase().includes(searchTerm.toLowerCase()))
+    member?.address?.toLowerCase().includes(searchTerm.toLowerCase()) ||
+    member?.user?.name.toLowerCase().includes(searchTerm.toLowerCase())
   );
 };
 
 const filterMemberByVerificationStatus = (
   member: Contributor | Watcher,
+  whitelistedAddresses: Address[],
   verificationType?: VerificationType,
 ) => {
+  const isVerified = isUserVerified(member.address, whitelistedAddresses);
+
   if (verificationType === VerificationType.Verified) {
-    return true;
-    // return member.isWhitelisted;
+    return isVerified;
   }
 
   if (verificationType === VerificationType.Unverified) {
-    return true;
-    // return !member.isWhitelisted;
+    return !isVerified;
   }
 
   return true;
-  // return !member.isWhitelisted;
 };
 
-export const filterMembers = <M extends Contributor | Watcher>(
-  members: M[],
+export const filterMembers = <Member extends Contributor | Watcher>(
```

Comment:
There's a `Member` type exported from `~types/graphql` that could replace the `Contributor | Watcher` part (in a few places in this file). On a side note, my impression was that a single letter is more or less a convention for generics, to avoid confusion with regular types

Reply:
Yep, changed it back. Agree that it is also the convention (which can be frustrating as it's harder to read).